_docs/development.md

```diff
@@ -25,23 +25,26 @@ You'll need the following to get started:
 ## Create temba user for PostgreSQL
 
 {% highlight bash %}
-$ createuser temba --pwprompt -d
+$ sudo apt-get install postgresql-client postgresql-server-dev-9.3 postgis*
+$ sudo -u postgres createuser temba --pwprompt -d
 Enter password for new role:
 Enter it again:
+$ sudo adduser temba
 {% endhighlight %}
 
 ## Create temba database, add PostGIS
 
 Create the database as temba user:
 {% highlight bash %}
-$ psql --user=temba postgres
+$ sudo -u temba psql --user=temba postgres
 postgres=> create database temba;
```

Comment:
Ya, I think we are leaning towards the 2nd option at the minute, as it covers people installing on OS X and everything else as well. Otherwise it is too much of a moving target.

You are going to have to be familiar with Linux to run RapidPro, no real way around that, this isn't Word. :)